### PR TITLE
fix: Update tutorial component style.module.less to lower the content height

### PR DESCRIPTION
**Pull request by chakrakan**
Addresses #834 

#### Before

Screen Shot 2022-03-03 at 9 19 12 PM

#### After

Screen Shot 2022-03-03 at 9 41 21 PM

### CODE REVIEW

**Comment by rschristian** [comment]
I think we need to correct this on all pages, not just the tutorial. The new banner height threw it off and covers some elements.

File: src/style/variables.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@header-height: 3.5rem; // 56px
+@header-height: 5rem; // 80px
```
Comment (rschristian) [inline src/style/variables.less:1]: Definitely don't want to do this, it increases the header size significantly.

Instead, increase the padding on the main content:

https://github.com/preactjs/preact-www/blob/b04ccc87e480c950a8a55b2a376d24dea09dda36/src/style/index.less#L93

```diff
-padding-top: @header-height;
+padding-top: calc(@header-height + 2.5rem);
```

**Review by rschristian — COMMENTED** [review]
Thanks for fixing this! Was just starting to look into it myself.

File: src/style/variables.less (hunk 1 above)
Comment (chakrakan) [reply src/style/variables.less]: updated!

File: src/style/variables.less [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @header-height: 3.5rem; // 56px
+@app-main-height: @header-height + 2.5rem; // 96px for banner
```
Comment (rschristian) [inline src/style/variables.less:2]: nit: I don't think this needs to be extracted to a variable, as it's only used in 1 place.

I do like having the comment there for explaining the extra `2.5rem` though, good idea.

**Comment by chakrakan** [comment]
This change should keep things consistent with the banner throughout `Tutorial` and `Guide` pages, which seemed to be the ones affected

### Guides 

Screen Shot 2022-03-04 at 1 30 34 AM

Screen Shot 2022-03-04 at 1 29 35 AM

File: src/style/variables.less (hunk 2 above)
Comment (chakrakan) [reply src/style/variables.less:2]: makes sense, I had pushed the change prior to seeing your comment, I'll use `calc` directly within `index.less`

Comment (rschristian) [reply src/style/variables.less:2]: Sounds good, thanks!

**Review by rschristian — APPROVED** [review]
Thanks for catching and fixing this so quickly!

**Pull request merged**